# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/ffvpx/libavcodec/raw.c
# Commit: 186e2a334f2d
# Full Hash: 186e2a334f2deb37b29f44b28bc8c83babeeb9a4
# Author: Paul Adenot <paul@paul.cx>
# Date: 2024-02-09 09:37:17
# Regressor Bug: 1877221
# File Overlap Count: 1
# Description:
#   Bug 1877221 - Pull in new ffmpeg code, reapply patch. r=media-playback-reviewers,chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D199909
# ==============================================================================

diff -r c5920de78d16 -r 186e2a334f2d media/ffvpx/libavcodec/raw.c
--- a/media/ffvpx/libavcodec/raw.c	Thu Feb 08 10:39:53 2024 +0000
+++ b/media/ffvpx/libavcodec/raw.c	Thu Feb 08 10:39:54 2024 +0000
@@ -206,6 +206,8 @@
     { AV_PIX_FMT_GBRAP10BE,    MKTAG(10 , 00 , '4', 'G') },
     { AV_PIX_FMT_GBRAP12LE,    MKTAG('G', '4', 00 , 12 ) },
     { AV_PIX_FMT_GBRAP12BE,    MKTAG(12 , 00 , '4', 'G') },
+    { AV_PIX_FMT_GBRAP14LE,    MKTAG('G', '4', 00 , 14 ) },
+    { AV_PIX_FMT_GBRAP14BE,    MKTAG(14 , 00 , '4', 'G') },
     { AV_PIX_FMT_GBRAP16LE,    MKTAG('G', '4', 00 , 16 ) },
     { AV_PIX_FMT_GBRAP16BE,    MKTAG(16 , 00 , '4', 'G') },
 